uri-scheme.md

```diff
@@ -0,0 +1,43 @@
+# URI Scheme
+## Scope
+This URI scheme describes the wormhole file transfer application. For other use cases of the wormhole spec and URI scheme has not been defined yet.
```

Comment:
These two read a little awkwardly to me.

What about, "This URI scheme pertains to the wormhole file-transfer application. A URI specification covering other use-cases of magic-wormhole has not yet been defined".